python/tstUtilities.py

```diff
@@ -626,7 +606,7 @@ def getUnitAndFunctionObjects(api, unitString, functionString):
                 returnUnitList.append(unitObject)
 
     # if coded mocks are enabled, and the unit name is an exact match, process the function name
-    if CODED_MOCK_ENABLED and len(returnUnitList) == 1:
+    if enviroSupportsMocking(api) and len(returnUnitList) == 1:
```

Comment:
Shouldn't there be a check way before this for enviroSupportsMocking?
For example, the extension should not be even calling the ct completion if the enviro does not support coded mocks
If we need this for the mock generator, then we should move it there